### PR TITLE
docs: add a document of inspect chaos experiment and revise a typo in other document

### PR DESCRIPTION
**Pull request by en-jin19**
Signed-off-by: en-jin19 <jinenwei@pingcap.com>

In this pr, I have:

- translated [#49](https://github.com/chaos-mesh/website-zh/pull/49)
- revised a typo in the document "simulate-aws-chaos.md”

### CODE REVIEW

File: docs/inspect-chaos-experiments.md [hunk 1]
```diff
@@ -1,5 +1,106 @@
...
 ---
 
-## TODO
+This document describes how to use Chaos Mesh to check running status and results of chaos experiments.
+
+## Introduction to steps of a chaos experiment
+
+In Chaos Mesh, the life cycle of a chaos experiment is divided into four steps, according to its running process:
+
+- Injecting: Chaos experiment is in the process of fault injection. Normally, this step lasts for a short time. If the "Injecting" step lasts a long time, it may be due to some exceptions in the chaos experiment. In this case, you can check `Events` to find the cause of the exceptions.
```
Comment (en-jin19) [inline docs/inspect-chaos-experiments.md:12]: do we need to add an example (e.g. estimated duration) for this part "Normally, this step lasts for a short time" to show the expected duration more clearly? @cwen0

Comment (cwen0) [reply docs/inspect-chaos-experiments.md:12]: The duration time depends on the chaos types and the account of the target pods. In most cases, if the duration time is more than 10s, the chaos may be abnormal.

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**